concerts/views.py

```diff
@@ -1,3 +1,122 @@
-from django.shortcuts import render
+import json
+import datetime
+import math
+
+from django.http         import JsonResponse
+from django.views        import View
+from django.db.models    import Q
+
+from concerts.models     import Concert,Location,ConcertSeat,Host,TicketingSite
+
+class ConcertUpcommingView(View):
+    def get(self, request):
+        concerts = Concert.objects.filter(date_performance__gt = datetime.datetime.now()).order_by('date_performance')
+        
+        if not concerts.exists():
+            return JsonResponse({"MESSAGE":"NO_UPCOMMING_CONCERTS"},status=404)
+
+        data = {
+                "concerts"          : [{
+                "pk"                : concert.id,
+                "title"             : concert.title,
+                "thumbnail_url"     : concert.thumbnail_url
+            } for concert in concerts]}
+        return JsonResponse(data, status=200)
```

Comment:
```suggestion
        data = {
                "concerts"          : [{
                    "pk"                : concert.id,
                    "title"             : concert.title,
                    "thumbnail_url"     : concert.thumbnail_url
            } for concert in concerts]}
        return JsonResponse(data, status=200)
```
더 하위에 생성되는 딕셔너리 이므로 시각적으로 분리해주시는 것이 가독성을 더 높여줍니다

Reply:
네!알겠습니다 수정했습니다!